\title{A pilot study examining the diet of introduced Alaska blackfish (\textit{Dallia pectoralis} T.\ H.\ Bean, 1880) in Kenai, Alaska, by metabarcoding}

\subtitle{\doi{10.7299/X78052XK}}

\author{by Matt Bowser\footnote{US Fish \& Wildlife Service, Kenai National Wildlife Refuge, Soldotna, Alaska, \email{matt\_bowser@fws.gov}}and Apphia Bowser}

\maketitle

\end{multicols}
\begin{figure}[H]
\begin{center}
%\vspace{2mm}
\includegraphics[width=\textwidth]{img/blackfish_pond.jpg}
\caption{Panoramic montage of a pond off of the Kenai Spur Highway and Candlelight Drive, the locality from which blackfish specimens were collected. A full resolution image is available on Arctos (\doi{10.7299/X7ZP46FP}).}
\label{blackfish_pond}
\end{center}
\end{figure} 
\begin{multicols}{2}   

\section{Introduction}

Last year we  wrote about some food items of the Alaska blackfish, \textit{Dallia pectoralis} T.\ H.\ Bean, 1880 \citep{Bowseretal2019},  a fish species that is native to most of Alaska, but not the Kenai Peninsula \citep{Eidametal2016, Bowser2018}. We wanted to learn more about how these introduced fish may alter the ecology of Kenai Peninsula waters, especially how blackfish may affect native fish species through competition for invertebrate prey.

\section{Methods}

We collected blackfish under Alaska Department of Fish \& Game permit Number SF2019-111.

On 23 August 2019 we collected blackfish from a small, shallow pond in Kenai, Alaska (60.5681~\textdegree{}N, -151.1901~\textdegree{}W $\pm$ 40~m) \citep{bowser2019}, the same pond from which we had obtained blackfish the previous year \citep{Bowseretal2019}. This pond (Figure \ref{blackfish_pond}) is fed by a small inlet stream and its level is maintained by a dam at the outlet, from which the stream flows through the Kenai Golf Course and into the Kenai River. There is little open water; most of the pond is thickly filled with \textit{Potamogeton} and flocculent iron bacterial scum. Only one other fish species, a single specimen of a nine-spined stickleback (\textit{Pungitius pungitius} (Linnaeus, 1758),  
\url{https://www.inaturalist.org/observations/31561030}), was observed in this pond.

We attempted to collect blackfish from other reaches of the stream below this pond where there would have been more potential for interaction between blackfish and other fish species, but found only small, juvenile blackfish downstream. 

The collected blackfish were placed on ice in a cooler, transported to the lab, and frozen. Later we thawed five adult blackfish (Arctos records \guid{KNWRObs:Fish:12}--\guid{KNWRObs:Fish:16}), measured their lengths, dissected out their entire guts, and squeezed gut contents into vials of UniGard -100 propylene glycol antifreeze.

Vials of gut contents were shipped to \acr{RTL} Genomics in Lubbock, Texas (\url{https://rtlgenomics.com/}) for RTL Genomics' standard microbial diversity assay using the \textit{mlCOIint}/\textit{jgHCO2198} (GGWACWGGWTGAACWGTWTAYCCYCC/TAIACYTCIGGRTGICCRAARAAYCA) primer set.

Extraction methods, sequencing methods, and resulting raw sequence data are provided in \citet{BowserBowser2020}.

\end{multicols}
\begin{figure}[H]
\begin{center}
%\vspace{2mm}
\includegraphics[width=\textwidth]{img/blackfish_phylo.pdf}
\caption{Phylogram of retained \acr{ESV} sequences. Colors representing major groups are the same as in Figure \ref{blackfish_pie_chart}. The tree can be viewed interactively or downloaded from \url{https://itol.embl.de/tree/16415961276811585237767}.}
\label{blackfish_phylo}
\end{center}
\end{figure} 
\begin{multicols}{2} 

Raw reads were processed using the \acr{SCVUC} \acr{COI} metabarcode pipeline version 4.3.0 (\url{https://github.com/Hajibabaei-Lab/SCVUC_COI_metabarcode_pipeline}). This pipline runs SeqPrep \citep{StJohn2016}, \acr{CUTADAPT} \citep{Martin2011}, \acr{VSEARCH} \citep{Rognes2016}, \acr{UNOISE} \citep{Edgar2016}, and the RDP classifier \citep{Wang2007} using the \acr{COI} Classifier v4 reference dataset \citep{PorterHajibabaei2018}. Processing steps were run via Snakemake \citep{KosterRahmann2012}. Our \acr{SCVUC} configuration file \citep{Bowser2020config} and snakefile \citep{Bowser2020snakefile} are available on Arctos.

The resulting exact sequence variants (\acr{ESV}s) were also compared to \acr{ESV}s obtained by \citet{Bowseretal2020} \citep[dataset: ][]{Bowseretal2020sup5}, sequences from an Alaska terrestrial arthropod \acr{DNA} barcode \acr{COI} reference library (\url{https://github.com/mlbowser/AKTerrInvCOILib}), and a \acr{FASTA} file of sequences from the authors' LifeScanner (\url{http://lifescanner.net/}) records (\url{http://www.boldsystems.org/index.php/Public_SearchTerms?query=DS-BOWSER}) using \verb|vsearch --usearch_global|. We also submitted our \acr{ESV}s to \acr{NCBI} \acr{BLAST} \citep{Johnsonetal2008} and the \acr{BOLD} \acr{ID} Engine \citep{Ratnasinghametal2007} searches  and scrutinized the results. We followed the guidlines of \citet{Sigovinietal2016} when assigning provisional names.

We removed all reads identified as \textit{Dallia pectoralis}; \textit{Bos taurus} Linnaeus, 1758; and all non-animals. The small numbers of \textit{Bos taurus} reads likely came from bovine serum albumin added during \acr{DNA} amplification. As a final check of identifications, we generated a phylogeny of the filtered \acr{ESV}s using NGPhylogeny.fr, ``NGPhylogeny Analyse - FastME/OneClick'' option \citep{DesperGascuel2002, CriscuoloGribaldo2010, JunierZdobnov2010, KatohStandley2013, Lefortetal2015, Lemoineetal2019} and examined the tree using i\acr{TOL} \citep{LetunicBork2019} (Figure \ref{blackfish_phylo}). The \acr{FASTA} file of retained \acr{ESV} sequences is available from Arctos \citep{Bowser2020bfdfas}. 


To prevent reporting false postive occurrences, we removed occurrences represented by $\leq 0.05\%$ of the total number of reads of an \acr{ESV}. Complete analysis details are provided in \citet{bowser2020}.

We tried to follow the guidelines of \citet{Penevetal2017} by publishing occurrence data on Arctos, which supplies occurrence data to \acr{GBIF}. Specimen records, images, and other related files have been made available via an Arctos project at \url{http://arctos.database.museum/project/10003367}.

\section{Results}

The retained 131 Exact Sequence Variants (Figure \ref{blackfish_phylo} ) were represented by 63,172 reads. The \acr{ESV}s were assigned to 103 uniquely identified food items and 137 occurrence records of these food items (Arctos records \guid{UAMObs:Ento:244406}--\guid{UAMObs:Ento:244542}). Arthropods represented by 62,166 (98\%) of the reads, followed by rotifers (431 reads, 0.7\%), annelid worms (384 reads, 0.6\%), molluscs (160 reads, 0.3\%), and one species of hydra (\textit{Hydra utahensis} Hyman, 1931, strain AK12b \textit{sensu} \citet{Martinezetal2010}, 31 reads, 0.05\%). The most abundant groups in terms of read abundances were odonates (32\%), dipterans (24\%), cladocerans (20\%), ostracods (16\%), and copepods (7\%) (Figure \ref{blackfish_pie_chart}).

\begin{figure}[H]
\begin{center}
\vspace{2mm}
\includegraphics[width=8cm]{img/blackfish_pie_chart.pdf}
\caption{Percentages of \acr{ESV} abundances in blackfish diet by taxonomic group.}
\label{blackfish_pie_chart}
\end{center}
\end{figure} 

Of the 103 unique identifications, 13 were comparatively abundunt, each representing $\geq 1$\% of the total number of reads (Figure \ref{blackfish_diet_items}). All of the reads of \textit{Aeshna eremita} Scudder, 1866 (Odonata: Aeshnidae), the most abundant species identified, came from a single blackfish. We detected \textit{Aeshna juncea} Linnaeus, 1758, the second most abundant species in our samples, from three fish. Ceratopogonidae sp.\ bfdZotu7 was both abundant and frequent in our samples, detected in gut contents of four out of five blackfish.

The relative abundance of each food item in terms of read abundances varied widely among the five blackfish individuals. For each fish, a different prey species was the most abundant food item.

Three of the most abundant \acr{ESV}s could be associated with niether described species nor \acr{BOLD} Barcode Index Numbers \citep{Ratnasinghametal2013}.  The \acr{ESV} identified as Ceratopogonidae sp.\ bfdZotu7 was 98.71\% similar ($p$-dist) to a private record on \acr{BOLD}. The \acr{ESV} tentatively identified as Cyprididae sp.\ bfZotu3 had no close matches in \acr{BOLD} or \acr{BAST}n search results, but the closest matches (83.99\% similarity) were Cyprididae. The \acr{ESV} identified as Podocopida sp.\ bfdZotu12 was closest (95.44\% similar) to a sequence from an ostracod specimen identified as Podocopida (\acr{BOLD} processid: \BOLD{OZFWC245-11}).

\end{multicols}
\begin{figure}[H]
\begin{center}
%\vspace{2mm}
\includegraphics[width=13cm]{img/blackfish_diet_items.pdf}
\caption{Read abundances of identified food items from each of five blackfish specimens. Only food items that represented $\geq 1$\% of the total number of reads were included. The area of each circle is proportional to read abundances.}
\label{blackfish_diet_items}
\end{center}
\end{figure} 
\begin{multicols}{2} 

Some of the \acr{ESV}s matched \acr{DNA} barcode sequences from locally collected entities that had not been associated with a formally described species. These included Lumbriculida sp.\ BOLD:ADR8620, a lumbriculid worm collected previously from near Nordic Lake, Soldotna (\acr{BOLD} processid: \BOLD{MOBIL6661-18}); \textit{Lumbriculus} sp.\ BOLD:AAG4731, another lumbriculid worm documented from a temporary pool in Soldotna (\acr{BOLD} processid: \BOLD{MOBIL1270-16}); and Trichoptera sp.\ SlikokOtu592, an \acr{ESV} from near Headquarters Lake documented by \citet{Bowseretal2020} (Arctos \acr{GUID}: \guid{UAMObs:Ento:239239}).

Seven chironomid species identified from our samples appeared to be new distribution records for Alaska. These were 
\textit{Chaetocladius conjugens} Brundin, 1947; 
\textit{Chironomus bifurcatus} Wuelker, Martin, Kiknadze, Sublette \& Michiels, 2009; 
\textit{Cladopelma bicarinata} (Brundin, 1947);
\textit{Cricotopus trifasciatus} (Meigen, 1813);
\textit{Dicrotendipes tritomus} (Thienemann \& Kieffer, 1916);
\textit{Orthocladius smolandicus} Brundin, 1947; and
\textit{Procladius nigriventris} (Kieffer, 1924).

\section{Discussion}

It appeared that the adult blackfish that we collected had recently consumed exclusively invertebrates, mostly arthropods. No \acr{DNA} from other fish species was detected. It should be noted, however, that other fish were comparatively rare in this pond. A single nine-spined stickleback was the only other fish documented. It may have been possible that juvenile blackfish were consumed by adult blackfish. These would not have been detected because all blackfish reads were removed from the analysis.

Overall, our results are consistent with other studies of blackfish diet \citep{OstdiekNardone1959, Chlupach1975, Gudkov1998, Eidam2015, Eidametal2016, Bowseretal2019} which collectively show that the most important prey groups
include cladocerans, ostracods, flies, dragonflies, snails, caddisflies,
and copepods. What separates our results from previous studies is that metabarcoding methods yielded much finer identifications, allowing us to document which species were consumed by blackfish. In previous studies, almost all identifications were coarse, with identifications lumped by orders or even higher-level groupings.

The variation in abundances of food items across the five blackfish individuals suggests that these fish are opportunistic, consuming whatever invertebrates they find and not seeking out any particular kind of prey item. It was surprising that we found none of the food items documented by \citet{Bowseretal2019} from blackfish from the same pond. Some differences in diet might have been expected due to season variation. \citet{Bowseretal2019} had collected blackfish on 18--19 October, two months later than our 23 August collecting date. Some of the food items documented by \citet{Bowseretal2019} were terrestrial wetland inhabitants that had likely become available to the blackfish due to flooding at the time. The water level of the pond was much lower when we sampled in August 2019 due to a warm, dry summer. Even with these differences in sampling date and water levels, we had expected to document at least some of the same species. The observed lack of overlap of observed prey items between the two studies supports our conclusion that blackfish are highly opportunistic.   

%Some of the most abundant food items were common species with widespread distributions. \textit{Candona candida} (O.F.Müller, 1776) is an ostracod with a worldwide distribution \citep{Delorme1970} that is found in a wide variety of freshwater habitat types \citep{Alkalajetal2019}. The cladoceran \textit{Eurycercus longirostris} Hann, 1982 is widespread in North America \citep{Bekkeretal2012}.

The rotifer \acr{ESV}s and other small-bodied invertebrates we observed may have been prey items of the blackfish or they may have been eaten by arthropods that were then eaten by blackfish. 

It should be noted that, due to potential biases related to metabarcoding methods, the relative read abundances that we report may not be directly related to the relative proportions of food items in the diets of the blackfish that we collected \citep[see][for an overview]{Deagleetal2018}. Regardless of potential metabarcoding biases due to differences in recovery and amplification of target \acr{DNA} across taxonomic groups, we believe that some of the differences in the wide range of read abundances that we observed had to do with how recently prey items had been consumed. Recent meals in blackfish stomachs would be expected to have more intact \acr{DNA} than the remains of food items further along in the intestines, where much of the \acr{DNA} would have been broken down.

In conclusion, we documented trophic relationships between Alaska blackfish and their prey at a particular time and place. To learn more about how blackfish interact with other fish species, we would like to see similar work done in waterbodies where there may be more interactions among fish species. It would also be good to examine diets from a wider range of sizes of blackfish as was done by \citet{Chlupach1975} and to compare blackfish diets with diets of other fish species in the same systems to learn more about potential competition and predation among fish species.

\section{Acknowledgments}

We thank Mike Baldwin for reviewing our list of potential new distribution records and pointing out that \textit{Angarotipula illustris} (Doane, 1901) was already known to occur in Alaska \citep{Brodo2018}. Rob Massengill, Kristine Dunker, and Derek Sikes provided comments that substantially improved the article.

%\bibliography{blackfish_diet}

\begin{thebibliography}{37}
\expandafter\ifx\csname natexlab\endcsname\relax\def\natexlab#1{#1}\fi
\expandafter\ifx\csname url\endcsname\relax
  \def\url#1{{\tt #1}}\fi
\expandafter\ifx\csname urlprefix\endcsname\relax\def\urlprefix{{\small URL}
  }\fi

\bibitem[{Bowser(2018)}]{Bowser2018}
Bowser, M.
\newblock 2018.
\newblock Kenai blackfish came from Bethel.
\newblock Refuge Notebook {\bfseries 20}:95--96.
\newblock
  \urlprefix\url{https://www.fws.gov/uploadedFiles/Region_7/NWRS/Zone_2/Kenai/Sections/What_We_Do/In_The_Community/Refuge_Notebooks/2018_Articles/Refuge_Notebook_v20_n47.pdf}.

\bibitem[{Bowser(2019)}]{bowser2019}
Bowser, M.~L.
\newblock 2019.
\newblock Work Journal 2019. Exported pdf version.
\newblock U.S. Fish \& Wildlife Service, Kenai National Wildlife Refuge.,
  Soldotna, Alaska.
\newblock \doi{10.7299/X7RB74XZ}.

\bibitem[{Bowser(2020{\natexlab{{\em a\/}}})}]{Bowser2020bfdfas}
Bowser, M.~L.
\newblock 2020{\natexlab{{\em a\/}}}.
\newblock Exact sequence variants from blackfish diet in FASTA format.
\newblock \doi{10.7299/X7V1253J}.

\bibitem[{Bowser(2020{\natexlab{{\em b\/}}})}]{Bowser2020config}
Bowser, M.~L.
\newblock 2020{\natexlab{{\em b\/}}}.
\newblock SCVUC pipeline configuration file for blackfish diet analysis.
\newblock \doi{10.7299/X7Q81DD8}.

\bibitem[{Bowser(2020{\natexlab{{\em c\/}}})}]{Bowser2020snakefile}
Bowser, M.~L.
\newblock 2020{\natexlab{{\em c\/}}}.
\newblock SCVUC pipeline snakefile for blackfish diet analysis.
\newblock \doi{10.7299/X7KH0NPJ}.

\bibitem[{Bowser(2020{\natexlab{{\em d\/}}})}]{Bowseretal2020sup5}
Bowser, M.~L.
\newblock 2020{\natexlab{{\em d\/}}}.
\newblock Supplementary material 5. ASV sequences.
\newblock \doi{10.3897/BDJ.8.e50124.suppl5}.

\bibitem[{Bowser(2020{\natexlab{{\em e\/}}})}]{bowser2020}
Bowser, M.~L.
\newblock 2020{\natexlab{{\em e\/}}}.
\newblock Work Journal 2020.
\newblock U.S. Fish \& Wildlife Service, Kenai National Wildlife Refuge.,
  Soldotna, Alaska.
\newblock \urlprefix\url{https://github.com/mlbowser/journal2020}.

\bibitem[{Bowser and Bowser(2020)}]{BowserBowser2020}
Bowser, M.~L., and A.~M. Bowser.
\newblock 2020.
\newblock {Raw metagenomic data from gut contents of \textit{Dallia pectoralis}
  specimens collected in Kenai, Alaska, in 2019}.
\newblock \doi{10.5281/zenodo.3731327}.

\bibitem[{Bowser et~al.(2019)Bowser, Bowser, Bowser, Bowser, and
  Bowser}]{Bowseretal2019}
Bowser, M.~L., M.~Bowser, E.~Bowser, A.~Bowser, and E.~Bowser.
\newblock 2019.
\newblock Some food items of introduced Alaska blackfish (\textit{Dallia
  pectoralis} T. H. Bean, 1880) in Kenai, Alaska.
\newblock Newsletter of the Alaska Entomological Society {\bfseries 12}:8--11.
\newblock \doi{10.7299/X7H995H7},
  \urlprefix\url{http://www.akentsoc.org/doc/AKES_newsletter_2019_n1_a04.pdf}.

\bibitem[{Bowser et~al.(2020)Bowser, Brassfield, Dziergowski, Eskelin, Hester,
  Magness, McInnis, Melvin, Morton, and Stone}]{Bowseretal2020}
Bowser, M.~L., R.~Brassfield, A.~Dziergowski, T.~Eskelin, J.~Hester, D.~R.
  Magness, M.~McInnis, T.~Melvin, J.~M. Morton, and J.~Stone.
\newblock 2020.
\newblock Towards conserving natural diversity: A biotic inventory by
  observations, specimens, DNA barcoding and high-throughput sequencing
  methods.
\newblock Biodiversity Data Journal {\bfseries 8}:e50124.
\newblock \doi{10.3897/BDJ.8.e50124}.

\bibitem[{Brodo(2018)}]{Brodo2018}
Brodo, F.
\newblock 2018.
\newblock Taxonomic review of \textit{Angarotipula} Savchenko, (Diptera:
  Tipulidae) in North America.
\newblock The Canadian Entomologist {\bfseries 150}:12–34.
\newblock \doi{10.4039/tce.2017.43}.

\bibitem[{Chlupach(1975)}]{Chlupach1975}
Chlupach, R.~S.
\newblock 1975.
\newblock Studies of introduced blackfish in waters of Southcentral Alaska.
\newblock Annual performance report for sport fish studies, volume 16, study
  G-II, Alaska Department of Fish and Game.
\newblock
  \urlprefix\url{http://www.sf.adfg.state.ak.us/FedAidPDFs/FREDF-9-7(16)G-II-K.pdf}.

\bibitem[{Criscuolo and Gribaldo(2010)}]{CriscuoloGribaldo2010}
Criscuolo, A., and S.~Gribaldo.
\newblock 2010.
\newblock BMGE (Block Mapping and Gathering with Entropy): a new software for
  selection of phylogenetic informative regions from multiple sequence
  alignments.
\newblock BMC Evolutionary Biology {\bfseries 10}:210.
\newblock \doi{10.1186/1471-2148-10-210}.

\bibitem[{Deagle et~al.(2019)Deagle, Thomas, McInnes, Clarke, Vesterinen,
  Clare, Kartzinel, and Eveson}]{Deagleetal2018}
Deagle, B.~E., A.~C. Thomas, J.~C. McInnes, L.~J. Clarke, E.~J. Vesterinen,
  E.~L. Clare, T.~R. Kartzinel, and J.~P. Eveson.
\newblock 2019.
\newblock Counting with DNA in metabarcoding studies: How should we convert
  sequence reads to dietary data?
\newblock Molecular Ecology {\bfseries 28}:391--406.
\newblock \doi{10.1111/mec.14734}.

\bibitem[{Desper and Gascuel(2002)}]{DesperGascuel2002}
Desper, R., and O.~Gascuel.
\newblock 2002.
\newblock Fast and accurate phylogeny reconstruction algorithms based on the
  minimum-evolution principle.
\newblock Journal of Computational Biology {\bfseries 9}:687--705.
\newblock \doi{10.1089/106652702761034136}.

\bibitem[{Edgar(2016)}]{Edgar2016}
Edgar, R.~C.
\newblock 2016.
\newblock UNOISE2: improved error-correction for Illumina 16S and ITS amplicon
  sequencing.
\newblock bioRxiv \doi{10.1101/081257},
  \urlprefix\url{https://www.biorxiv.org/content/early/2016/10/15/081257}.

\bibitem[{Eidam(2015)}]{Eidam2015}
Eidam, D.~M.
\newblock 2015.
\newblock Trophic ecology of introduced populations of Alaska blackfish
  (\textit{Dallia pectoralis}) in the Cook Inlet Basin, Alaska.
\newblock Master's thesis.
\newblock \urlprefix\url{https://pqdtopen.proquest.com/pubnum/1587621.html}.

\bibitem[{Eidam et~al.(2016)Eidam, von Hippel, Carlson, Lassuy, and
  L{\'o}pez}]{Eidametal2016}
Eidam, D.~M., F.~A. von Hippel, M.~L. Carlson, D.~R. Lassuy, and J.~A.
  L{\'o}pez.
\newblock 2016.
\newblock Trophic ecology of introduced populations of Alaska blackfish
  (\textit{Dallia pectoralis}) in the Cook Inlet Basin, Alaska.
\newblock Environmental Biology of Fishes {\bfseries 99}:557--569.
\newblock \doi{10.1007/s10641-016-0497-6}.

\bibitem[{Gudkov(1998)}]{Gudkov1998}
Gudkov, P.~K.
\newblock 1998.
\newblock Bering Sea \textit{Dallia pectoralis} in the Chukchi Peninsula.
\newblock Journal of Ichthyology {\bfseries 38}:199--203.

\bibitem[{Johnson et~al.(2008)Johnson, Zaretskaya, Raytselis, Merezhuk,
  McGinnis, and Madden}]{Johnsonetal2008}
Johnson, M., I.~Zaretskaya, Y.~Raytselis, Y.~Merezhuk, S.~McGinnis, and T.~L.
  Madden.
\newblock 2008.
\newblock {NCBI BLAST: a better web interface}.
\newblock Nucleic Acids Research {\bfseries 36}:W5--W9.
\newblock \doi{10.1093/nar/gkn201}.

\bibitem[{Junier and Zdobnov(2010)}]{JunierZdobnov2010}
Junier, T., and E.~M. Zdobnov.
\newblock 2010.
\newblock The Newick utilities: high-throughput phylogenetic tree processing in
  the UNIX shell.
\newblock Bioinformatics {\bfseries 26}:1669--1670.
\newblock \doi{10.1093/bioinformatics/btq243}.

\bibitem[{Katoh and Standley(2013)}]{KatohStandley2013}
Katoh, K., and D.~M. Standley.
\newblock 2013.
\newblock MAFFT multiple sequence alignment software version 7: Improvements in
  performance and usability.
\newblock Molecular Biology and Evolution {\bfseries 30}:772--780.
\newblock \doi{10.1093/molbev/mst010}.

\bibitem[{Köster and Rahmann(2012)}]{KosterRahmann2012}
Köster, J., and S.~Rahmann.
\newblock 2012.
\newblock {Snakemake---a scalable bioinformatics workflow engine}.
\newblock Bioinformatics {\bfseries 28}:2520--2522.
\newblock \doi{10.1093/bioinformatics/bts480}.

\bibitem[{Lefort et~al.(2015)Lefort, Desper, and Gascuel}]{Lefortetal2015}
Lefort, V., R.~Desper, and O.~Gascuel.
\newblock 2015.
\newblock FastME 2.0: A comprehensive, accurate, and fast distance-based
  phylogeny inference program.
\newblock Molecular Biology and Evolution {\bfseries 32}:2798--2800.
\newblock \doi{10.1093/molbev/msv150}.

\bibitem[{Lemoine et~al.(2019)Lemoine, Correia, Lefort, Doppelt-Azeroual,
  Mareuil, Cohen-Boulakia, and Gascuel}]{Lemoineetal2019}
Lemoine, F., D.~Correia, V.~Lefort, O.~Doppelt-Azeroual, F.~Mareuil,
  S.~Cohen-Boulakia, and O.~Gascuel.
\newblock 2019.
\newblock NGPhylogeny.fr: new generation phylogenetic services for
  non-specialists.
\newblock Nucleic Acids Research {\bfseries 47}:W260--W265.
\newblock \doi{10.1093/nar/gkz303}.

\bibitem[{Letunic and Bork(2019)}]{LetunicBork2019}
Letunic, I., and P.~Bork.
\newblock 2019.
\newblock {Interactive Tree Of Life (iTOL) v4: recent updates and new
  developments}.
\newblock Nucleic Acids Research {\bfseries 47}:W256--W259.
\newblock \doi{10.1093/nar/gkz239},
  \urlprefix\url{https://doi.org/10.1093/nar/gkz239}.

\bibitem[{Martin(2011)}]{Martin2011}
Martin, M.
\newblock 2011.
\newblock Cutadapt removes adapter sequences from high-throughput sequencing
  reads.
\newblock EMBnet.journal {\bfseries 17}:10--12.
\newblock \doi{10.14806/ej.17.1.200},
  \urlprefix\url{https://journal.embnet.org/index.php/embnetjournal/article/view/200}.

\bibitem[{Martínez et~al.(2010)Martínez, Iñiguez, Percell, Willner,
  Signorovitch, and Campbell}]{Martinezetal2010}
Martínez, D., A.~Iñiguez, K.~Percell, J.~Willner, J.~Signorovitch, and
  R.~Campbell.
\newblock 2010.
\newblock Phylogeny and biogeography of \textit{Hydra} ({Cnidaria}: {Hydridae})
  using mitochondrial and nuclear {DNA} sequences.
\newblock Molecular Phylogenetics and Evolution {\bfseries 57}:403--410.
\newblock \doi{10.1016/j.ympev.2010.06.016},
  \urlprefix\url{http://www.sciencedirect.com/science/article/pii/S1055790310002873}.

\bibitem[{Ostdiek and Nardone(1959)}]{OstdiekNardone1959}
Ostdiek, J.~L., and R.~M. Nardone.
\newblock 1959.
\newblock Studies on the Alaskan blackfish \textit{Dallia pectoralis} I.
  Habitat, size and stomach analyses.
\newblock The American Midland Naturalist {\bfseries 61}:218--229.
\newblock \urlprefix\url{http://www.jstor.org/stable/2422353}.

\bibitem[{Penev et~al.(2017)Penev, Mietchen, Shravan Chavan, Hagedorn,
  Stuart Smith, Shotton, Éamonn Ó~Tuama, Senderov, Georgiev, Stoev,
  John Groom, Remsen, and C. Edmunds}]{Penevetal2017}
Penev, L., D.~Mietchen, V.~Shravan Chavan, G.~Hagedorn, V.~Stuart Smith,
  D.~Shotton, Éamonn Ó~Tuama, V.~Senderov, T.~Georgiev, P.~Stoev,
  Q.~John Groom, D.~Remsen, and S.~C. Edmunds.
\newblock 2017.
\newblock Strategies and guidelines for scholarly publishing of biodiversity
  data.
\newblock Research Ideas and Outcomes {\bfseries 3}:e12431.
\newblock \doi{10.3897/rio.3.e12431}.

\bibitem[{Porter and Hajibabaei(2018)}]{PorterHajibabaei2018}
Porter, T.~M., and M.~Hajibabaei.
\newblock 2018.
\newblock Automated high throughput animal {CO1} metabarcode classification.
\newblock Scientific Reports {\bfseries 8}:4226.
\newblock \doi{10.1038/s41598-018-22505-4},
  \urlprefix\url{https://doi.org/10.1038/s41598-018-22505-4}.

\bibitem[{Ratnasingham and Hebert(2007)}]{Ratnasinghametal2007}
Ratnasingham, S., and P.~D.~N. Hebert.
\newblock 2007.
\newblock {BOLD}: {The} {Barcode} of {Life} {Data} {System}
  (www.barcodinglife.org).
\newblock Molecular Ecology Notes {\bfseries 7}:355--364.
\newblock \doi{10.1111/j.1471-8286.2007.01678.x}.

\bibitem[{Ratnasingham and Hebert(2013)}]{Ratnasinghametal2013}
Ratnasingham, S., and P.~D.~N. Hebert.
\newblock 2013.
\newblock A {DNA}-based registry for all animal species: {The} {Barcode}
  {Index} {Number} ({BIN}) system.
\newblock PLoS ONE {\bfseries 8}:e66213.
\newblock \doi{10.1371/journal.pone.0066213}.

\bibitem[{Rognes et~al.(2016)Rognes, Flouri, Nichols, Quince, and
  Mahé}]{Rognes2016}
Rognes, T., T.~Flouri, B.~Nichols, C.~Quince, and F.~Mahé.
\newblock 2016.
\newblock {VSEARCH}: a versatile open source tool for metagenomics.
\newblock PeerJ {\bfseries 4}:e2584.
\newblock \doi{10.7717/peerj.2584}.

\bibitem[{Sigovini et~al.(2016)Sigovini, Keppel, and
  Tagliapietra}]{Sigovinietal2016}
Sigovini, M., E.~Keppel, and D.~Tagliapietra.
\newblock 2016.
\newblock Open Nomenclature in the biodiversity era.
\newblock Methods in Ecology and Evolution {\bfseries 7}:1217--1225.
\newblock \doi{10.1111/2041-210X.12594}.

\bibitem[{St.~John(2016)}]{StJohn2016}
St.~John, J.
\newblock 2016.
\newblock SeqPrep.
\newblock \urlprefix\url{https://github.com/jstjohn/SeqPrep}.

\bibitem[{Wang et~al.(2007)Wang, Garrity, Tiedje, and Cole}]{Wang2007}
Wang, Q., G.~M. Garrity, J.~M. Tiedje, and J.~R. Cole.
\newblock 2007.
\newblock Na{\"\i}ve Bayesian classifier for rapid assignment of rRNA sequences
  into the new bacterial taxonomy.
\newblock Applied and Environmental Microbiology {\bfseries 73}:5261--5267.
\newblock \doi{10.1128/AEM.00062-07},
  \urlprefix\url{https://aem.asm.org/content/73/16/5261}.

\end{thebibliography}
