\title{Miller Creek Watershed aquatic invertebrate inventory}

\author{by
 Matthew L.\ Bowser\footnoteremember{KenaiNWR}{USFWS \href{https://www.fws.gov/refuge/kenai/}{Kenai National Wildlife Refuge}, Soldotna, Alaska}\footnote{\email{matt\_bowser@fws.gov}},
 Samuel I.\ Artaiz\footnoterecall{KenaiNWR},
 Jake M.\ Danner\footnoterecall{KenaiNWR},
 Kris K.\ Dent\footnoteremember{SoldotnaADFG}{\href{https://www.adfg.alaska.gov/}{Alaska Department of Fish and Game}, Soldotna, Alaska},
 Robert L.\ Massengill\footnoterecall{SoldotnaADFG},
 Benjamin Meyer\footnoteremember{KWF}{\href{https://kenaiwatershed.org/}{Kenai Watershed Forum}, Soldotna, Alaska},
 Dom Watts\footnoterecall{KenaiNWR}, 
 Chelsea Wisotzkey\footnoterecall{KWF}, and
 Warren R.\ Wyrick\footnoterecall{SoldotnaADFG}
 
 \parbox[t][][s]{\columnwidth}{\vspace{3em}} % This box exists just to prevent the authors from breaking over to a new column.
 }

\maketitle

\end{multicols}

\vspace{-1cm}
\begin{center}
 \parbox[t][][s]{14cm}{\section{Abstract}
Because benthic macroinvertebrates and zooplankton are susceptible to the pesticide rotenone, surveys of freshwater macroinvertebrates were conducted in the Miller Creek Watershed, Kenai Peninsula, Alaska ahead of a planned rotenone treatment in fall 2021. Currently, 32 of 32 planned samples have been collected in 2021. Another 32 post-treatment invertebrate samples are planned in 2022 to enable comparison of pre- and post-treatment freshwater invertebrate communities.}
 
\end{center}

\vspace{4mm}

\begin{multicols}{2}

\section{Introduction}

Invasive northern pike (\textit{Esox lucius} Linnaeus, 1758) were discovered in the Miller Creek watershed in 2019 \citep{KNWR2021} and a rotenone treatment to eradicate pike from the drainage was planned for fall 2021.

Benthic macroinvertebrates and zooplankton are susceptible to rotenone \citep{Finlaysonetal2018}, some of them dying at lower rotenone concentrations than fish \citep{Rach1988}. Planktonic crustations decline rapidly after rotenone treatments, generally taking months to years to fully recover \citep{Kiseretal1963, Anderson1970}. In other southcentral Alaska lakes, most pre-treatment plankton species were present in lake water with in one to two years of treatment and plankton abundace returned to pre-treatment levels in the third year \citep{Chlupach1977}. On the Kenai Peninsula, most invertebrate taxa that were present in lakes before present were again present one year after treatment \citep{Massengill2014, Massengill2017}. 

\citet{Finlaysonetal2018} recommended sampling aquatic invertebrates before and after rotenone treatments to document survival and recovery of these animals.

Freshwater bioassessments have been based on morphological identfications, but metabarcoding has been found to be an efficient means of conducting freshwater biodiversity surveys \citep{Turunenetal2021}.

For this paper we sought to

\begin{enumerate}
\item document the pre-treatment aquatic invertebrate communities in the water bodies to be treated with rotenone as recommended by \citet{Finlaysonetal2018} and demonstrated by \citep{Massengill2014, Massengill2017},
\item test metabarcoding as a method for conducting this kind of routine inventory and monitoring, and
\item compare the results of a metabarcoding analysis pipeline provided as a commercial service with the results of using a pipeline featuring Je \citep{Girardotetal2016}, MetaWorks \citep{PorterHajibabaei2020}, and local reference libraries.
\end{enumerate}

% Quote below from Massengill (2014).
%In Southcentral Alaska, the effect of rotenone to aquatic invertebrate communities is typically temporary in nature and usually requires 1â€“3 years for posttreatment levels of zooplankton to be restored to pretreatment levels (Chlupach 1977). This is longer than reported in many other areas of North America where invertebrate recovery often takes a year or less (Kiser et al. 1963; Hamilton et al. 2009). Other studies show that zooplankton such as cladocerans and copepods have rotenone resistant eggs capable of reseeding a lake after a rotenone treatment (Bradbury 1986; Melaas et al. 2001). Fall applications may help zooplankton communities recover because many species are in rotenone-resistant life stages and there is time for population recovery before spring (Melaas et al. 2001).

\section{Methods}

We began collecting a small number of invertebrate specimens in the Miller Creek watershed after northern pike and then elodea were discovered in the drainage. A small number of additional samples were collected in 2020.

A more thorough inventory effort began in 2021. We generally followed the methods used in previous aquatic inventories that were performed before and after rotenone applications on the Kenai Peninsula \citep{Massengill2014, Massengill2017} with the exception that most identifications are to be obtained by metabarcoding instead of morphological identifications.

Aquatic mollusks were sampled under Alaska Department and Fish and Game Aquatic Resource Permit No.\ SF2021-134.

The rotenone treatment of these three connected water bodies was completed on October 4--7, 2021.

\subsection{Study area}

The waterbodies surveyed were those in which rotenone will be applied, the same water bodies in which northern pike have been detected. These include North Vogel Lake, Vogel Lake, and Miller Creek. North Vogel Lake (14~ha) and Vogel Lake (49~ha) are separated by about 200~m and connected by a small stream that flows through a wetland. Both lakes are similar in terms of water quality metrics \citep{Meyer2021}. Miller Creek drains Vogel Lake, running approximately 7~km to Cook Inlet.

\subsection{Field sampling}

We generally followed the methods of \citet{Massengill2014, Massengill2017} for lake sampling aquatic invertebrates. Each of the three waterbodies was to be surveyed once in July and once in August. Sampling was to be conducted in the same temporal windows in 2021 and 2022.

At North Vogel Lake, Vogel Lake, and upper Miller Creek we sampled twice in 2021: first on July 20--23 and second on August 28. We sampled at 15 sites using three methods. We failed to make it out to the mouth of Miller Creek in July and August, collecting samples there only on September 13, 2021. 

Sample sizes were chosen to be similar to those used by \citet{Massengill2014, Massengill2017}. \citet{Massengill2014}, to survey invertebrates in 38~ha Scout Lake, collected 3 light trap samples, 5 Ekman samples, 5 D-net samples, and 2 Wisconsin net samples. For 157~ha Stormy Lake, \citet{Massengill2014, Massengill2017} collected 5 Ekman samples, 5 D-net samples, and 3 Wisconsin net sample.

\input{Miller_Creek_inventory_table_design}

At each of two visits per year we collected 3 D-net samples, 3 Ekman dredge samples, and 2 Wisconsin net samples in Vogel Lake; 2 D-net samples, 2 Ekman dredge samples, and 1 Wisconsin net sample in North Vogel Lake; and 3 D-net samples in Miller Creek, a total of 16 invertebrate samples per visit (Table \ref{tab:nsamples}), 32 samples per year, and 64 samples over the two year project.

In addition to the 64 samples intended to be processed by metabarcoding, invertebrates were collected opportunistically while the authors were working within the study area.

Field methods generally followed \citet{Massengill2014, Massengill2017}. We took vertical plankton tows in the deepest parts of the lakes using an Aquatic Research Instruments Wisconsin net. We sampled littoral areas using with D-nets. We obtained benthic samples using either an AMS Incorporated model 445.11 Ekman dredge or an AMS Incorporated model 445.60  stainless steel dredge. Most benthic samples were sorted using a series of sieves.

Samples were collected into ethanol or SK picglobal 99.9\% pure propylene glycol. Samples intended for metabarcoding were collected into propylene glycol or, in the case of two samples, collected into ethanol and then transferred to propylene glycol.

\subsection{Specimen processing}

Specimens were deposited in the Kenai National Wildlife Refuge's entomology and invertebrate collections (international collection coden: \acr{KNWR}) and managed with Arctos (\url{https://arctos.database.museum/}).

For the morphologically identified specimens, we used the following references: \cite{Borroretal1989}, \cite{Brooks1957}, \cite{Collet2008}, \cite{Durfee2005}, \cite{Haneyetal2013}, \cite{Hatch1953}, \cite{Kenner2009}, \cite{Marx1957}, \cite{MerrittCummins1996}, \cite{Merrittetal2008}, \cite{Reid1987}, \cite{Rileyetal2002}, \cite{Roughley2000}, \cite{Wallis1933}, and \cite{White1983}.


\subsection{Molecular methods}

Metabarcoding samples were stored in a -23~\textdegree{}C freezer exept when samples were being processed. Invertebrates were separated from debris by hand under a dissecting microscope. Care was taken to reduce possible cross-contamination of \acr{DNA} among samples. Samples were shipped out on ice on September 29, 2021, arriving the next day at \acr{MR DNA} (Shallowater, Texas, \url{http://www.mrdnalab.com}).

We chose to use the \textit{mlCOIintF}/\textit{jgHCO2198} (\acr{G\-G\-W\-A\-C\-W\-G\-G\-W\-T\-G\-A\-A\-C\-W\-G\-T\-W\-T\-A\-Y\-C\-C\-Y\-C\-C}\-/\-\acr{T\-A\-I\-A\-C\-Y\-T\-C\-I\-G\-G\-R\-T\-G\-I\-C\-C\-R\-A\-A\-R\-A\-A\-Y\-C\-A}) primer set of \citet{Lerayetal2013} for \acr{PCR}, targeting a 313~bp region of the \acr{COI} \acr{DNA} barcoding region. This primer set has been successfully used for a wide variety of invertebrates, including terrestrial invertebrates \citep{Bowseretal2020}, freshwater macroinvertebrates \citep{Hajibabaeietal2019}, freshwater plankton \citep{Yangetal2017}, and stomach contents of freshwater fish \citep{BowserBowser2020}.

The \textit{mlCOIintF}/\textit{jgHCO2198} primer pair was used with barcode on the forward primer in a 30--35 \acr{PCR} using the HotStarTaq Plus Master Mix Kit (Qiagen, \acr{USA}) under the following conditions: 94~\textdegree{}C for 3 minutes, followed by 30--35 cycles of 94~\textdegree{}C for 30~s, 53~\textdegree{}C for 40 seconds and 72~\textdegree{}C for 1~minute, after which a final elongation step at 72~\textdegree{}C for 5~minutes was performed.  After amplification, PCR products were checked in 2\% agarose gel to determine the success of amplification and the relative intensity of bands. Multiple samples were pooled together in equal proportions based on their molecular weight and \acr{DNA} concentrations. Pooled samples were purified using calibrated Ampure XP beads. The pooled and purified \acr{PCR} product was used to prepare an illumina \acr{DNA} library. Sequencing was performed at \acr{MR DNA} on a MiSeq following the manufacturerâ€™s guidelines. 

\subsection{MR DNA analysis pipeline}

Sequence data were processed by using \acr{MR DNA} analysis pipeline (\acr{MR DNA}, Shallowater, Texas).  In summary, sequences were joined, sequences < 150~bp were removed, and sequences with ambiguous base calls were removed. Sequences were quality filtered using a maximum expected error threshold of 1.0 and dereplicated. The dereplicated or unique sequences were denoised and chimeras were removed. Final zero-radius operational taxonomic units were taxonomically classified using \acr{BLAST}n against a curated database derived from \acr{NCBI} (\url{www.ncbi.nlm.nih.gov}).   

\subsection{Je + MetaWorks pipeline}

We processed the raw sequence data on the \acr{USGS} Yeti supercomputer \citep{USGSARC2021} using R, version 4.1.1 for manipulating data and Je, version 2.0.RC \citep{Girardotetal2016} for demultiplexing. The raw data included reads in alternating directions with the sample barcodes only on one read. Accordingly, we exectued the \verb|je demultiplex| command, accepting the defaults that only require one of the two reads to contain a sample barcode.

Raw read data were processed using MetaWorks, version 1.9.5 \citep{PorterHajibabaei2020}, running the default analysis options for metazoan \acr{COI} \acr{DNA} barcode sequences. Sequences were identified using the \acr{RDP} classifier, version 2.13 \citep{Wangetal2007} and the CO1 Classifier, version 4.0.1 reference library \citep{Porter2017, PorterHajibabaei2018}. 

We processed data in R, versions 4.0.1 and 4.1.2 \citep{RCoreTeam2021} using the R packages Biostrings, version 2.62.0 \citep{Pagesetal2021} bold, version 1.2.0 \citep{Chamberlain2021}; and reshape2, version 1.4.4 \citep{Wickham2007}.

We assembled a reference library by concatenating final Exact Sequence Variant (\acr{ESV}) sequences from \citet{Bowseretal2020, BowserBowser2020} and the DS-BOWSER project on BOLD\footnote{\url{https://boldsystems.org/index.php/Public_SearchTerms?searchMenu=records&query=DS-BOWSER}}, then dereplicating them with \verb|vsearch --derep_fulllength|. We looked for matches with this reference library using \verb|vsearch --usearch_global|.

As in \citet{Massengill2014, Massengill2017}, the metric will be simple presence or absence of taxa within the lakes before and after treatment.

\section{Results}

Specimen data and images are available via an Arctos project (\url{https://arctos.database.museum/project/10003613}). Specimen records are available via a search for records from this project (\url{https://arctos.database.museum/SpecimenResults.cfm?project_id=10003613}). All Arctos records from this project are also published to the Global Biodiversity Information Facility (\acr{GBIF}, \url{https://www.gbif.org/}) via the VertNet Integrated Publishing Toolkit (\url{http://ipt.vertnet.org/}). ``Research grade'' iNaturalist observations are also published to \acr{GBIF}.

Georeferenced photo observations have been gathered in a project on iNaturalist.org at \url{https://www.inaturalist.org/projects/miller-creek-invertebrate-inventory}.

A corresponding record was also made on ServCat (\url{https://ecos.fws.gov/ServCat/Reference/Profile/139305}). Raw metabarcoding data are available from Servcat (\url{https://ecos.fws.gov/ServCat/Reference/Profile/139306}).

\end{multicols}

\begin{longtable}{p{3in}p{1.2in}p{1.2in}p{1.2in}}
\caption{Invertebrates collected excluding metabarcoding samples.}\label{tableids}\\

\hline
\hline
\textbf{Identification}&&&\\ 
\hspace{0em}Kingdom&&&\\
\hspace{0.8em}Phylum&&&\\
\hspace{1.6em}Class& \textbf{Miller Creek} & \textbf{North Vogel Lake} & \textbf{Vogel Lake}\\
\hspace{2.4em}Order&&&\\
\hspace{3.2em}Family&&&\\
\hspace{4.0em}Subfamily, etc.\ &&&\\
\hline
\endfirsthead

\hline
\textbf{Identification} & \textbf{Miller Creek} & \textbf{North Vogel Lake} & \textbf{Vogel Lake}\\
\hline
\endhead

\multicolumn{4}{c}{\textit{continued on next page\ldots}}\\
\hline
\endfoot

\hline
\hline
\endlastfoot

\input{Miller_Creek_inventory_table_ids}

\end{longtable}

\begin{multicols}{2}



\section{Discussion}

\textit{Donacia hirticollis} Kirby, 1837 had not previously been documented from the Kenai National Wildlife Refuge.

\section{Acknowledgments}

We thank \href{https://www.inaturalist.org/}{iNaturalist.org} people \verb|amr_mn| and \verb|zvkemp| (Zach Kemp) for identifications.

\bibliography{Miller_Creek_inventory}


